src/com/snake/and/ladder/Game.java

```diff
@@ -0,0 +1,115 @@
+package com.snake.and.ladder;
```

Comment:
This class is doing a lot of things. Should be broken into at least two classes where one of them should control the flow including scanning data.